src/manifolds/Euclidean.jl

```diff
@@ -132,6 +132,42 @@ embed(::Euclidean, p)
 
 embed!(::Euclidean, q, p) = copyto!(q, p)
 
+@doc raw"""
+    embed(::EmbeddedManifold{𝔽,Euclidean{𝔽},Euclidean{𝔽},ET}, p)
+
+A [`Euclidean`](@ref) `M` manifold can be embedded into a [`Euclidean`](@ref) `N`
+if the length of the array dimension and representation size is elementwise less or equal.
+The remainder is filles with zeros for sure.
+"""
+function embed(
+    M::EmbeddedManifold{𝔽,Euclidean{n,𝔽},Euclidean{m,𝔽2},ET},
```

Comment:
Actually, shouldn't this be defined for all embedded manifolds, not just this one? After all it's just the generic allocate-then-mutate function...

Reply:
Wait, it's already in ManifoldsBase: https://github.com/JuliaManifolds/ManifoldsBase.jl/blob/7471d948b4e5a0485cff823313233fb2397c485b/src/ManifoldsBase.jl#L260

Reply:
Well, nearly. The one in base allocates an object of size of `M.manifold` somehow, not of `M.embedding`, that's what this one fixes.

Reply:
Oh, I think it's a bug in ManifoldsBase then (and should be fixed there).

Reply:
Should we leave this here until its fixed?

Reply:
I can quickly fix it in base.

Reply:
Embedding should work with ManifoldsBase 0.9.2 without this method 🙂

Reply:
Right, I changed that.